src/diff/manager.py

```diff
@@ -1,25 +1,39 @@
-from src.diff.diff import Diff, DiffFormat, DiffReport, DiffError
-import src.diff.hashcash as hashcash
+# from src.diff.diff import Diff, DiffFormat, DiffReport, DiffError
+# import src.diff.hashcash as hashcash
+from diff import Diff, DiffFormat, DiffReport, DiffError
+import hashcash as hashcash
 import os
+import glob
+import pickle
+# Manager 기능 = directory path를 입력하면, directory path가 실제 testing을 진행할 대상의 프로젝트의 디렉터리 패스고 그걸 입력하면 거기에 있는 모든 파일에 대해서 cash가 있는지 확인하고 잇으면 비교하고, 없으면 전부 newline으로 때리고, cache를 만드는 것까지 매니저의 기능이다. recursive하게 파일 뒤지고 이런걸.. 하다가 말았다. 
 
+#수요일까지 
 
 class Manager:
-    def __init__(self, dir_path: str, rel_cache_path: str):
+    def __init__(self, dir_path: str, rel_cache_path: str = "/.cache"):
         """
         An instance of diff management module.
         :param dir_path: Absolute path of target path to analyze.
         :param rel_cache_path: Relative path of cache files. (default = ./.cache)
         """
         self.dir_path = dir_path
-        self.cache_path = os.path.join(dir_path, rel_cache_path)
+        # self.cache_path = os.path.join(dir_path, rel_cache_path)
+        # 윈도우에서는 os.path.join이 이상하게 먹히네요..
+        self.cache_path = dir_path + rel_cache_path
 
-    def __analyze_file(self, file_path: str) -> DiffFormat:
+    def analyze_file(self, file_path: str) -> DiffFormat:
         # todo: remove empty newlines
         # todo: remove comments
-        rel_path = os.path.relpath(file_path, self.dir_path)
-        f = open(file_path, 'r')
+        # file_path = root is self.dir_path.
+        rel_path = self.dir_path+"./"+file_path
```

Comment:
```suggestion
        rel_path = self.dir_path + os.sep + file_path
```
Maybe this will work both for linux and windows?